Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -30,7 +30,7 @@ TARGET_PATH="$BUILD_BUCKET_PATH/content/packs"
 PACKS_FULL_TARGET_PATH="$GCS_BUILD_BUCKET/$TARGET_PATH"
 BUCKET_FULL_TARGET_PATH="$GCS_BUILD_BUCKET/$BUILD_BUCKET_PATH"
 echo "Copying master files at: $SOURCE_PATH to target path: gs://$PACKS_FULL_TARGET_PATH ..."
-gsutil -m cp -r "gs://$GCS_MARKET_BUCKET/$SOURCE_PATH" "gs://$PACKS_FULL_TARGET_PATH"
+gsutil -m cp -r "gs://$GCS_MARKET_BUCKET/$SOURCE_PATH" "gs://$PACKS_FULL_TARGET_PATH" > "$CIRCLE_ARTIFACTS/logs/Prepare Content Packs For Testing.log" 2>&1
```

Comment:
are you sure you want to remove this from the circle log and only save to file?
if not, use `tee` please

Reply:
Yes, I'm sure.this is pure debug prints - 99% of the times they are not interesting at all, and they make this step console pass the 40,000 chars limit which after that the logs colors are disabled.